code/modules/antagonists/swarmer/swarmer.dm

```diff
@@ -482,14 +482,6 @@
 	return new fabrication_object(loc)
 
 /mob/living/simple_animal/hostile/swarmer/proc/Integrate(atom/movable/target)
-	if(isobj(target))
```

Comment:
Please do not hard remove code. Comment it out with a skyrat change tag attached. Also this is not a good fix as it lacks essential checks which would do more damage than it fixes.

Reply:
I'll just close the PR for now , since DeltaFire made a better fix for it.I'll keep in mind for other PR's. I don't think the check is that necesarry since any object you are not supposed to eat will just not have a integrate amount .